src/utils/generateFlowDiagramData.js

```diff
@@ -97,14 +98,25 @@ const generateFlowDiagramData = (data) => {
     { flowData: {}, mostRecentYear: -Infinity, mostRecentMonth: -Infinity }
   );
 
-  Object.values(flowData).forEach((item) => {
-    const hint = item.isNotAvailable
+  Object.values(flowData).forEach((metric) => {
+    const hint = metric.isNotAvailable
       ? null
-      : generateHint(mostRecentYear, mostRecentMonth, item.item);
+      : generateHint(mostRecentYear, mostRecentMonth, metric.item);
 
     if (hint) {
       // eslint-disable-next-line no-param-reassign
-      item.hint = hint;
+      metric.hint = hint;
+    }
+
+    const { item } = metric;
```

Comment:
Why do we need to initialize a separate `item` for comparison below? (And why did we need to change from `item` to `metric` in the first place?)

Reply:
With help of utility function we don't need this change at all, so rollback